packages/logger/src/tests/logger.ts

```diff
@@ -0,0 +1,44 @@
+import { NSlogger } from '..';
+import { lorem } from 'faker';
+
+const mockDate = new Date(0);
+jest.spyOn(global, 'Date').mockImplementation(() => mockDate as any); // TypeScript workaround
+const consoleSpy = jest.spyOn((console as any)._stderr, 'write').mockImplementation();
+
+const message = lorem.sentence();
+const params: Record<string, unknown>[] = [
+  { [lorem.word()]: { [lorem.word()]: lorem.sentence() }, [lorem.word()]: lorem.sentence() },
+  { [lorem.word()]: { [lorem.word()]: lorem.sentence() } },
+];
+
+const logger = NSlogger('test');
+
```

Comment:
Am I missing something but wasn't there a main `describe` block? If not could you please add it to wrap our hooks & `it` cases here. Can be just `Logger basic tests` or whatever you'd like to call them